Get getNotificationsStatus interval from config

adriendupuis opened the pull request:
| Question      | Answer
| ------------- | ---
| Tickets       | N/A
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)
<!-- Keep in mind: Your contribution has to be compatible with GPL-2.0 as well: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLModuleLicense -->


Get `getNotificationsStatus` (`notifications/count`) interval from YAML config parameters.

One could want (or need) to increase this interval to reduce server activity.

Doc could be needed to explain that this is set in milliseconds but I couldn't find any doc for other parameters like `.timeout`.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

adriendupuis committed:
Get getNotificationsStatus interval from YAML config

adriendupuis commented on src/bundle/Resources/public/js/scripts/admin.notifications.modal.js:
```diff
@@ -179,9 +179,11 @@
 
     const notificationsTable = modal.querySelector(SELECTOR_TABLE);
     currentPageLink = notificationsTable.dataset.notifications;
+    let interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval);
+    if (!interval) { interval = INTERVAL; }
```
Fallback on hardcoded `INTERVAL` in case of `NaN` or `0`.

Steveb-p commented on src/bundle/Resources/public/js/scripts/admin.notifications.modal.js:
```diff
@@ -179,9 +179,11 @@
 
     const notificationsTable = modal.querySelector(SELECTOR_TABLE);
     currentPageLink = notificationsTable.dataset.notifications;
+    let interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval);
+    if (!interval) { interval = INTERVAL; }
```
```suggestion
    const interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval) || INTERVAL;
```

dew326 replied:
I like this, just one small thing
`const interval = parseInt(notificationsTable.dataset.notificationsCountInterval, 10) || INTERVAL;`

adriendupuis replied:
Thank you @dew326 for suggesting to add the radix/base.

adriendupuis committed:
admin.notifications.modal.js: rework interval

`interval` as a const with its value assigned on 1 code line

Co-authored-by: Paweł Niedzielski <pawel.tadeusz.niedzielski@gmail.com>

adriendupuis committed:
admin.notifications.modal.js: parseInt radix

“this does not default to 10!” — https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt
